src/model/model.py

```diff
@@ -1,11 +1,112 @@
 from skimage import morphology
 import numpy as np
-from src.utils import protector
+import nibabel as nib
+import imageio
+import os
+import pickle
+import imageio.core.util
+
+
+# remove imageio warnings
+def silence_imageio_warning(*args, **kwargs):
+    pass
+
+
+imageio.core.util._precision_warn = silence_imageio_warning
 
 
 class Model:
+    def load(self, path):
+        """
+        this method has a vocation to load any type of file
+
+        Parameters
+        ----------
+        path: str
+            handled extensions: nii, nii.gz, png, jpg, txt, pkl
+
+        Return
+        ------
+        data: any type of data
+
+        """
+        root, ext = os.path.splitext(path)
+        if ext == '.txt':
+            with open(path, 'r') as f:
+                data = f.read()
+        elif ext == '.pkl':
+            with open(path, 'rb') as f:
+                data = pickle.load(f)
```

Comment:
Just to be curious, in which use case do you think to use pickle files?

Reply:
You can save any type of data in pickle file, 
for instance if ou want to save an image which contains infinite values you can't with png or jpg but you can use pickle.. 
But it's temporary while we find another best solution. Do you think I should removed it now?

Reply:
I don't think that it is a problem to leave it. It is just that it's not a usual image format, only people knowing python and pickle know it, but it is not really common. I agree to keep it, I was just curious.

Reply:
ok !